packages/registry/openapi3.yaml

```diff
@@ -0,0 +1,157 @@
+openapi: 3.0.1
```

Comment:
I have a general feeling that we overuse uuid v4.
I think namsepaces can be easily a sequential number.

wdyt? what about having services ids as numbers too?

Reply:
for `namespace_id` I agree this might simplify things.
as for `service_id` due to the fact that actions such as service rotation require a human user to post the request, having a number as an id is prone to human error.